.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "phpcs.standard": "phpcs.xml.dist"
```

Comment:
Is this required for one of the `PHPCS` extensions for VS Code? Both `phpcs.xml` and `phpcs.xml.dist` are some of the default filenames resolved by the phpcs runner:

> If you run PHP_CodeSniffer without specifying a coding standard, PHP_CodeSniffer will look in the current directory, and all parent directories, for a file called either .phpcs.xml, phpcs.xml, .phpcs.xml.dist, or phpcs.xml.dist

https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#using-a-default-configuration-file

and:

> phpcs.autoConfigSearch [ Scope: All | Optional | Type: boolean | Default: true ]
>
> Automatically search for any phpcs.xml, phpcs.xml.dist, phpcs.ruleset.xml or ruleset.xml file to use as configuration. Overrides phpcs.standard configuration when a ruleset is found.

https://marketplace.visualstudio.com/items?itemName=ikappas.phpcs

Reply:
Naw, I've just always set my workspace setting up like that. I forgot to add it to the `.gitignore`